catalog/ui/catalog-ui-search/src/main/webapp/component/workspaces-templates/workspaces-templates.less

```diff
@@ -143,19 +115,15 @@
   height: ~'calc(100vh - 2*@{minimumLineSize})';
   overflow: auto;
 
-  .home-templates-header-hint,
-  .home-templates-header-button {
-    display: none;
-  }
 }
 
 .has-footer @{customElementNamespace}workspaces-templates.is-expanded,
 .has-header @{customElementNamespace}workspaces-templates.is-expanded {
-  height: ~'calc(100vh - 2*@{minimumLineSize} - 1.5rem)';
+  height: @minimumLineSize + 2*@minimumButtonSize + 2*@minimumSpacing + 2*@minimumSpacing + 3*@minimumButtonSize;
```

Comment:
❓ These were dynamic before, should they really be fixed now?

Reply:
For this I just wanted the "open" position to be the same height as what the original "closed" position was, so I replaced the original expression from [line 5](https://github.com/codice/ddf/pull/2820/files/74b377cbb2ebfa2e41631adef42260e3f437fc6d#diff-71477352d216c2d89179a531c19cf092L5) here